### ISSUE TITLE
RGBD data

### ISSUE THREAD

**Issue by tungyen** [Omni6DPose/GenPose2#5]
If the input is RGB image and the partial point cloud from the depth image, is this input acceptable for GenPose?

**Comment by Jiyao06** [comment]
Sure, GenPose takes as input the detection and segmentation results of a single object, along with the corresponding partial point cloud.

**Comment by tungyen** [comment]
Thank you for the reply. And I also want to ask if GenPose++ is able to operate on unknown data.

**Comment by Jiyao06** [comment]
Yes, GenPose++ can generalize to unknown objects, and we will release a convenient version of GenPose++ with SAM for the downstream tasks.

**Comment by youngfriday** [comment]
@Jiyao06 Thanks for yur great work! I'm looking forward to seeing the convenient version of GenPose++ with SAM. Can I ask for the releasing date?

**Comment by Jiyao06** [comment]
Thank you for your interest. We will release this version in early April.

**Comment by Moshu413** [comment]
> 感谢您的关注。我们将在 4 月初发布此版本。

Thank you for your work. Could you please share the planned release timeline?
